logbook/ticketing.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 """
-    logbook.ticketing
+logbook.ticketing
```

Comment:
No made a mistake, sorry! Editor messed it up.
On 25/09/2015 7:28 PM, "Rotem Yaari" <notifications@github.com> wrote:

> In logbook/ticketing.py
> <https://github.com/mitsuhiko/logbook/pull/152#discussion_r40460879>:
>
> > @@ -1,6 +1,7 @@
> >  # -*- coding: utf-8 -*-
> >  """
> > -    logbook.ticketing
> > +logbook.ticketing
>
> Is this intentional?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mitsuhiko/logbook/pull/152/files#r40460879>.
>